### ISSUE TITLE
Noise reduction preview differs from exported file

### ISSUE THREAD

**Issue by blitzgneisserin** [Beep6581/RawTherapee#5608]
Apparently there is more noise in the 100% preview than in the exported file. I have tested this with several photos, in certain photos it is very pronounced. In other photos it is barely visible.

To reproduce:

1. Open a raw with a lot of noise.
2. Apply noise reduction. Set Gamma to a low value, e.g. 1 or 1.2, set luma noise reduction to a moderately high value, e.g. 25-50, set recover details to a high value, e.g. 80
3. export the photo
4. compare the results

Screenshots:

![screenshot_rt_noise_ff](https://user-images.githubusercontent.com/16564210/72270121-f2e46b80-3624-11ea-81db-9ed1110d235c.png)
![screenshot_rt_noise_rt](https://user-images.githubusercontent.com/16564210/72270124-f37d0200-3624-11ea-95cd-32108afa0747.png)

Here is the link to the raw + sidecare file.
https://drive.google.com/open?id=10z5Cgn9V6tZPgs7I15wcX1h7Ft__dt9C

RT development build 13 January 2020 + RT 5.7 from Debian repo
OS Debian Bullseye
screen: BenQ SW240 (110% AdobeRGB, hardware calibrated+profiled)

**Comment by Thanatomanic** [comment]
I can confirm. First, RT 200% view of my edit (Neutral + Exposure + some NR)

![image](https://user-images.githubusercontent.com/6567747/72275815-f41a9600-362e-11ea-9cc1-a0d29d985385.png)

Then the export, enlarged to 200%, same area:
![image](https://user-images.githubusercontent.com/6567747/72275830-0399df00-362f-11ea-9fbc-f2f99156a7fe.png)

**Comment by Thanatomanic** [comment]
In addition: even without touching the gamma slider, there is a noticeable difference in the 100% preview and the export. For good measure I exported to png to avoid jpeg artifacts.

Possibly related: #4773

**Comment by heckflosse** [comment]
Does it also happen when luma noise reduction is set to zero?

**Comment by Thanatomanic** [comment]
No, I don't think so. The output seems identical with the preview in that case.

**Comment by blitzgneisserin** [comment]
> Possibly related: #4773

I have seen that too, though it seemed to me that the report was not clear enough. And it was reported some time ago...
